src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -145,7 +145,7 @@ class ActivitiesService(
 
   private fun toNomisActivity(schedule: ActivitySchedule, activity: Activity): CreateActivityRequest {
     return CreateActivityRequest(
-      code = "$activity.id-$schedule.id",
+      code = "${activity.id}-${schedule.id}",
```

Comment:
bug found by new tests

Reply:
The code is displayed in Nomis - I think we need to work out something better than this? (maybe that's a new ticket?)